README.rst

```diff
@@ -54,6 +54,44 @@ Basic Usage
 
     print(indexes)
 
+Exceptions handling
+---------------
+
+Since 5.0.0 release, we introduced children classes for base HttpError class,
+for example:
+```
+class PayloadTooLarge(HttpError):
+    pass
```

Comment:
Including `pass` here (and in subsequent examples) seems inappropriate. Perhaps just list the exception types and a brief description for each?